etc/lmod-setup.csh

```diff
@@ -34,6 +34,9 @@ else if ( "$L_MACHINE" == odin ) then
    module --initial_load --no_redirect restore
    setenv MODULEPATH "/oldscratch/ywang/external/hpc-stack/modulefiles/mpi/intel/2020/cray-mpich/7.7.16:/oldscratch/ywang/external/hpc-stack/modulefiles/compiler/intel/2020:/oldscratch/ywang/external/hpc-stack/modulefiles/core:/oldscratch/ywang/external/hpc-stack/modulefiles/stack:/opt/cray/pe/perftools/21.02.0/modulefiles:/opt/cray/ari/modulefiles:/opt/cray/pe/craype-targets/default/modulefiles:/opt/cray/pe/modulefiles:/opt/cray/modulefiles:/opt/modulefiles"
 
+else if ( "$L_MACHINE" == wcoss2 ) then
+   module reset
+
```

Comment:
@chan-hoo - This file is problematic from a design point of view because it is unscalable/unmaintainable/unportable/untestable.  This stuff should be moved to some kind of individual machine-specific configuration file.  But, since you are just following the existing template, I can't argue with this change right now.  But this should be flagged as something that needs rethinking with respect to the overall design.

Reply:
@christopherwharrop-noaa, thank you for the comment. If you still think that it is better to separate two module commands and move envvar to build_wcoss2_intel, please let me know. I'll talk about this with my EMC colleagues.